linkcheck/bookmarks/chrome.py

```diff
@@ -26,7 +26,7 @@ def get_profile_dir():
     """Return path where all profiles of current user are stored."""
     if os.name == 'nt':
         if "LOCALAPPDATA" in os.environ:
-            basedir = unicode(os.environ["LOCALAPPDATA"], nt_filename_encoding)
+            basedir = unicode(os.environ["LOCALAPPDATA"], nt_filename_encoding)  # noqa: F821
```

Comment:
We don't support Python 2 any more, and I'm pretty sure os.environ contains strings on Windows.
```suggestion
            basedir = os.environ["LOCALAPPDATA"]
```

Reply:
Good enough for me.